pom.xml

```diff
@@ -37,13 +37,13 @@
   </modules>
 
   <properties>
-    <openMRSVersion>2.4.2</openMRSVersion>
-    <openMRSRuntimeVersion>2.4.2</openMRSRuntimeVersion>
+    <openMRSVersion>2.7.7</openMRSVersion>
+    <openMRSRuntimeVersion>2.7.7</openMRSRuntimeVersion>
```

Comment:
I really wish we could be consistent about this across all of the module we have upgraded.  We should not just take the latest released version from the 2.7.x line, because in config.xml we (typically) use this property to indicate what version of OpenMRS is required, and we don't strictly _need_ to require 2.7.7, we can simply require 2.7.0 (or some other, consistent, minimum version).  Whatever we do, we should do it the same across all refapp module so that we can confidently say that all will work as of some specific version.

Reply:
I downgraded this to 2.7.0